contributing_to_docs/doc_guidelines.adoc

```diff
@@ -629,98 +629,123 @@ Text for admonition
+
+- Pod
+- Deployment
+- Operator
+- DaemonSet (and not "daemon set", "daemonset", or "Daemonset")
+
 == Quick Markup Reference
 
 |===
 |Convention |Markup |Example rendered output
 
 |Code blocks
-a|....
-Use the following syntax for the `oc` command:
+
+a|Use the following syntax for the `oc` command:
 
 ----
 $ oc <action> <object_type> <object_name_or_id>
 ----
-....
 
 a|Use the following syntax for the `oc` command:
 
 ----
 $ oc <action> <object_type> <object_name_or_id>
 ----
 
-|Inline commands, operations, literal values, variables, parameters, settings,
-flags, environment variables, and user input.
+a|Use backticks for all non-GUI "system items", including:
 
-In general (when inline within sentences), something that either operates or
-acts on something else, or when referencing some item from a file or code block /
-example.
-a|$$`oc get`$$
+* Inline commands, operations, literal values, variables, parameters, settings,
+flags, environment variables, user input
+* System term/item, user names, unique or example names for individual API
+objects/resources (e.g., a Pod named "mypod"), daemon, service, or software
+package
+* RPM packages
+* File names or directory paths
 
-$$`GET`$$
+a|$$`oc get`$$
 
 $$Set the `upgrade` variable to `true`.$$
 
+$$Use the `--amend` flag.$$
+
 $$Answer by typing `Yes` or `No` when prompted.$$
 
-$$Use the `--amend` flag.$$
+$$`user_name`$$
+
+$$`service_name`$$
 
-a|Use the `oc get` command to get a list of services that are currently defined.
+$$`package_name`$$
 
-The `GET` operation can be used to do something.
+$$`filename`$$
+
+a|Use the `oc get services` command to get a list of services that are currently defined.
+
+Use the `--amend` flag.
 
 Set the `upgrade` variable to `true`.
 
 Answer by typing `Yes` or `No` when prompted.
 
-Use the `--amend` flag.
+`cluster-admin` user
+
+`firewalld` service
+
+`rubygems` RPM package
+
+The `express.conf` configuration file is located in the `/usr/share` directory.
 
 |System or software variable to be replaced by the user
 a|$$`<project>`$$
 
 $$`<deployment>`$$
 
-a|
-Use the following command to roll back a deployment, specifying the deployment name:
+a|Use the following command to roll back a Deployment, specifying the Deployment name:
 
 `oc rollback <deployment>`
 
-|System term/item, user names, unique or example names for individual API objects/resources (e.g., a pod named "mypod"), GUI menu items and buttons, daemon, service, or software package.
-
-In general (when inline within sentences), something that can be operated or
-acted upon, or unique or example names for system terms/items in general.
-
-a|$$*system item*$$
+|Use single asterisks for web console / GUI items (menus, buttons, page titles, etc.).
 
-$$*daemon*$$
+a|Choose $$*Cluster Console*$$ from the drop-down menu.
 
-$$*service*$$
+Navigate to the $$*Operators -> Catalog Sources*$$ page.
 
-$$*software package*$$
+Click the $$*Create Subscription*$$ button.
```

Comment:
s/Click the $$*Create Subscription*$$ button./Click $$*Create Subscription*$$.
IBM Style says to just call out the button label or hovertext and omit the word "button."  There are a couple of instances of this - please update them all.